lib/Display/BitmapRegion.cpp

```diff
@@ -21,13 +22,34 @@ void BitmapRegion::render(GxEPD2_GFX* display) {
     Serial.print(F("WARN - tried to render bitmap file that doesn't exist: "));
     Serial.println(variableValue);
   } else {
+    this->previousBound = this->boundingBox;
+    uint16_t new_width, new_height;
+    if(getBitmapDimensions(variableValue, &new_width, &new_height)){
+      this->boundingBox.w = new_width;
+      this->boundingBox.h = new_height;
+    }
+
+    // In case it's a new bitmap with a different size.
+    Rectangle clearingBounds = getBoundingBox();
+    Serial.println("sup");
+    Serial.println(clearingBounds.w);
+    Serial.println(clearingBounds.h);
```

Comment:
Let's nuke these unless it's important info (if it is, should format it and use `printf_P`).

```suggestion
```